### PR TITLE
Added gupnp-tools 0.8.10 (new formula)

### PR DESCRIPTION
**Pull request by ened**
Provides a collection of developer tools utilising GUPnP and GTK+.

### CODE REVIEW

File: Library/Formula/gupnp-tools.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "gtk+3"
+  depends_on "gtksourceview3"
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-debug", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/gupnp-tools.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/gupnp-tools.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require "formula"
```
Comment (mikemcquaid) [inline Library/Formula/gupnp-tools.rb]: This isn't needed.

**Comment by mikemcquaid** [comment]
Please fix the `brew audit --strict` warnings: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27478/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_gupnp_tools___strict/

File: Library/Formula/gupnp-tools.rb (hunk 1 above)
Comment (ened) [inline Library/Formula/gupnp-tools.rb]: Difficult - as the formula is for a GUI tool. Suggestion?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `gupnp-tools 0.8.10 (new formula)`.

**Comment by bfontaine** [comment]
Just saw this, isn’t it the same formula as the one in [homebrew-x11](https://github.com/Homebrew/homebrew-x11/blob/master/gupnp-tools.rb)?

**Comment by DomT4** [comment]
^ Yup, looks like it. We can remove it from `x11` and remove any reference to migrating it in `tap_migrations`.

It was probably only moved to X11 in the first place because GTK+3 originally had an X11 backend (It no longer does). Either that or the X11 dependency is actually optional, in which case it's cool to stay in the core as well.

**Comment by DomT4** [comment]
^ Fixed by https://github.com/Homebrew/homebrew-x11/commit/b6b0c180fa4dbd3612622467a931e4b24fabed08 and https://github.com/Homebrew/homebrew/commit/411c2178448916d4005c059bc35ee780b5af53f5

**Pull request closed without merging**